### DISCUSSION TITLE
오늘의 문제의 풀이 제한시간 관련해 문의드립니다!

### DISCUSSION THREAD

**Discussion by CoodingPenguin** [tony9402/baekjoon#203] [Question & Answer] [answered]
안녕하세요, 
우선 이 레포로 코딩테스트 준비하는데 정말 도움이 많이 되고 있어 감사하다는 말씀부터 드립니다.

다름이 아니라 오늘의 문제로 실제 코딩테스트를 보는 것처럼 연습을 해보려는데 오늘의 문제의 제한 시간은 어느 정도로 생각하면 될까요?

감사합니다!

**Comment by tony9402** [comment, marked as answer]
안녕하세요.

문제 난이도는 쉬움, 보통, 보통, 어려움 순으로 뽑게 되어있습니다.
해당 난이도를 생각해본다면 2시간 ~ 3시간 정도가 적당할 것으로 예상됩니다.

처음엔 3시간 정도 제한을 두고 푸시는 게 나을 것 같습니다.

**Reply by CoodingPenguin** [reply]
답변 감사드립니다!
